src/gov/nasa/worldwind/wms/WMSTiledImageLayer.java

```diff
@@ -145,7 +145,6 @@ public static AVList wmsGetParamsFromCapsDoc(WMSCapabilities caps, AVList params
         setFallbacks(params);
 
         // Setup WMS URL builder.
-        params.setValue(AVKey.WMS_VERSION, caps.getVersion());
```

Comment:
This is already done at line 130, IF the params does not contain a value already. This line just overwrites any WMS_VERSION that might be set in configuration.